packages/openbox.rb

```diff
@@ -0,0 +1,56 @@
+require 'package'
+
+class Openbox < Package                 # The first character of the class name must be upper case
+  description 'Openbox is a highly configurable, next generation window manager with extensive standards support.'
+  homepage 'http://openbox.org/'
+  version '3.6.1'
+  compatibility 'all'
+  source_url 'http://openbox.org/dist/openbox/openbox-3.6.1.tar.xz'
+  source_sha256 'abe75855cc5616554ffd47134ad15291fe37ebbebf1a80b69cbde9d670f0e26d'   # Use the command "sha256sum"
+
+  depends_on 'libev'
+  depends_on 'startup_notification'
+  depends_on 'xcb_util_cursor'
+  depends_on 'xcb_util_keysyms'
+  depends_on 'xcb_util_wm'
+  depends_on 'xcb_util_xrm'
+  depends_on 'yajl'
+  depends_on 'wayland' => :build
+  depends_on 'sommelier'
```

Comment:
Can we make `depends_on 'sommelier'` the last dependency line?

Reply:
Done